### ISSUE TITLE
how to use in objective c project?

### ISSUE THREAD

**Issue by xiasenhai** [gmertk/BusyNavigationBar#7]
really awesome project, but how to integrate with objective-c project?

**Comment by gmertk** [comment]
Thanks @xjiaoyang. Are you using Cocoapods?

**Comment by xiasenhai** [comment]
@gmertk  yes, i use cocoapods, and i installed the pod but i can not import BusyNavigationBar header file. 
i even want translate this project to objective-c, but i dont yet have much time. please help me, thanks!

BY THE WAY REALLY AN AWESOME PROJECT! :+1: 
that's why i really want to use in my project!
